README.md

```diff
@@ -4,22 +4,65 @@ A service which helps you integrate your upstream GitHub project into Fedora
 ecosystem. Think of it as [packit](https://github.com/packit-service/packit) on
 steroids.
 
+## Introductory description
+With packit, it's easy to integrate your upstream project into Fedora Project.
+Your pull requests are built in Fedora version of your choice,
+so you're always sure that your software works. When you're ready for a release,
+packit takes care of all the hassle and is able to ship your new upstream release
+into Fedora Rawhide.
+
+### Easy to use
+
+Using Packit Service is very straightforward: add one config file to your repository,
+followed by an RPM spec file and you're good to go. We also support packit as a CLI tool,
+so you can always try things locally on your own.
+
+### Makes you confident
+
+Packit Service validates your pull requests by building your software in Fedora OS.
+Once the builds are done, packit lets you know how to install your change
+inside your environment. Get confidence by trying out your changes yourself
+before shipping them to your users.
+
+## Steps how to install Packit-as-a-Service into your projects or organizations
+
+For the installation packit into your project there are available two approaches
+
+### Install from Packit-as-a-Service page
+1. From the [Packit-service page](https://github.com/organizations/packit-service/settings/apps/packit-as-a-service) page,
```

Comment:
I don't think people will be able to access this link. At least I can't in a private window. We should make the app public.